au.org.access-nri/model/experiment/reproducibility/checksums/README.md

```diff
@@ -1,9 +1,8 @@
-# ACCESS-OM2 Checksums
+# Model Configuration Checksums
+
+This schema is used for checksum files used as part of model reproducibility CI checks for model configurations. For example, the checksums are saved to released ACCESS-OM2 model configurations in the [ACCESS-NRI/access-om2-configs](https://github.com/ACCESS-NRI/access-om2-configs) repository. The pytests that are run as part of these CI tests that generate and compare checksums are in the [ACCESS-NRI/model-config-tests](https://github.com/ACCESS-NRI/model-config-tests/) repository.
+
```

Comment:
I think we should add a sentence saying something along the lines that checksums were the original target, and are still preferred, but in some cases formatted numbers are used.

Reply:
Thanks, I've added that to the readme